docs/Custom-Brimcap-Config.md

```diff
@@ -306,12 +309,12 @@ export LD_LIBRARY_PATH="/usr/local/lib"
 
 As we did with Zeek and Suricata, we create a [wrapper script](https://github.com/brimdata/brimcap/blob/main/examples/nfdump-wrapper.sh) to act as our
 Brimcap analyzer. It works in two phases, first creating binary NetFlow records
-and then converting them to CSV. `nfpcapd` only accepts a true pcap file input
+and then converting them to NDJSON. `nfpcapd` only accepts a true pcap file input
```

Comment:
When I ran the latest `nfdump` today for the first time in a while, it seems they recently mucked up their CSV support (they now output one of their fields onto a separate line, which made Zed's CSV reader choke), but the good news is that they added an NDJSON output format that I'm more than happy to recommend instead. I'd only been using `nfdump`'s CSV output because their JSON output was always a giant array in the past which meant bumping into https://github.com/brimdata/zed/issues/3865. Now that they can output NDJSON (which they call `json-log` since it's more compatible with log-centric tools like Splunk and Logstash) we can take advantage of that.